### PR TITLE
Added Portuguese translation

### PR DESCRIPTION
**Pull request by promoreira**
Just a small contribution!!

### CODE REVIEW

**Comment by agateau** [comment]
Hey, sorry for the late review. Looking into it!

File: android/assets/ui/languages.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <Language id="ru" name="Русский" fontSet="xolonium"/>
     <Language id="bn" name="বাংলা" fontSet="noto-bengali"/>
     <Language id="zh_CN" name="简体中文" fontSet="noto-cjk"/>
+    <Language id="pt_BR" name="PortugueseBR" fontSet="xolonium-kwajong"/>
```
Comment (agateau) [inline android/assets/ui/languages.xml:12]: The text in the `name` attribute is the one shown in the language list. You can see it in this screenshot:
screenshot-2022-09-10-100053 052

This text should be written in the language of the translation itself, can you change it to say "Brazilian Portuguese" in Brazilian Portuguese?

File: android/assets/screens/credits.gdxui [hunk 2]
```diff
@@ -89,6 +89,9 @@
                 <Label>Chinese:</Label>
                 <Label>Lu Xu</Label>
                 <Label/>
+                <Label>PortugueseBR:</Label>
```
Comment (agateau) [inline android/assets/screens/credits.gdxui:92]: The text here is shown in the credits screen:
screenshot-2022-09-10-100711 256

It should say something like "Brazilian Portuguese".

**Review by agateau — CHANGES_REQUESTED** [review]
Looks good overall, just a few minor things to change:
- No need to commit the `pt_BR.mo` file, can you remove it?
- It seems you have renamed the pl.po file to pt_BR.po: this removed the Polish translation :/ Can you restore it?

**Comment by agateau** [comment]
I am planning to release Pixel Wheels 0.23.0 at the end of October / beginning of November. Do you think you can remaining issues? I would love to include your work in the next release!

**Comment by agateau** [comment]
No activity here, I have to close the PR, sorry. Feel free to open a new one if you resume work on it.

**Comment by Wilker-uwu** [comment]
i don't remember much how it works on github. how do i pick up this pull request to continue? no promisses that i'll actually finish. i just wanna know this.

**Pull request closed without merging**